cli/cmd/create_secret.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"errors"
+	"github.com/keptn/keptn/cli/pkg/credentialmanager"
+	"github.com/spf13/cobra"
+)
+
+type createSecretCmdParams struct {
+	Data  []string
+	Scope *string
+}
+
+var createSecretParams *createSecretCmdParams
+
+var createSecretCommand = &cobra.Command{
+	Use:          `secret SECRETNAME --from-literal="key1=value1"" --from-literal="key2=value2" --scope=my-scope`,
```

Comment:
I find this easier to read:
```suggestion
	Use:          `secret SECRET_NAME --from-literal="key1=value1"" --from-literal="key2=value2" --scope=my-scope`,
```